src/Symfony/Component/Process/ExecutableFinder.php

```diff
@@ -75,10 +66,17 @@ public function find(string $name, string $default = null, array $extraDirs = []
             foreach ($dirs as $dir) {
                 if (@is_file($file = $dir.\DIRECTORY_SEPARATOR.$name.$suffix) && ('\\' === \DIRECTORY_SEPARATOR || @is_executable($file))) {
                     return $file;
+                } elseif(basename($dir) === $name.$suffix && @is_executable($dir)) {
```

Comment:
missing space after elseif
+ doesn't this miss a `!@is_dir($dir)` check?

Reply:
Fixed